files/image_config/coredump/coredump-config.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+DISABLE_COREDUMP_CONF="/etc/sysctl.d/50-disable-coredump.conf"
+
+if [ "$(redis-cli -n 4 HGET "COREDUMP|config" "enabled")" = "false" ] ; then
```

Comment:
If key is present && carry value as false, then we disable. In other words the default behavior is "enabled=true". To disable, one has to create this key explicitly.

Instead, why not require a key  for disabling, which would imply default is enabled.

"COREDUMP|disabled" == true

Reply:
It is common place to see configuration knobs with a positive intent. So if we flip this around, it may confuse the user with other usages of true/false kind of configurations.